manifests/j/JetBrains/DataSpell/2022.3.2/JetBrains.DataSpell.locale.en-US.yaml

```
@@ -0,0 +1,25 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: JetBrains.DataSpell
+PackageVersion: 2022.3.2
+PackageLocale: en-US
+Publisher: JetBrains s.r.o.
+PublisherUrl: https://www.jetbrains.com
+PublisherSupportUrl: https://www.jetbrains.com/data-tools
+PrivacyUrl: https://www.jetbrains.com/privacy-security
+PackageName: JetBrains DataSpell
```

```suggestion
PackageName: DataSpell
```
Must match Control Panel / Apps&Features